connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java

```diff
@@ -152,6 +152,66 @@ public void testCredentialDelta() throws ApiException, IOException {
     session.stop();
   }
 
+  @Test
+  public void testDeleteDeltaTable() throws ApiException, IOException {
+    createCommonResources();
+    SparkSession session = createSparkSessionWithCatalogs(SPARK_CATALOG, CATALOG_NAME);
+
+    String loc0 = "s3://test-bucket0" + generateTableLocation(SPARK_CATALOG, DELTA_TABLE);
+    setupExternalDeltaTable(SPARK_CATALOG, DELTA_TABLE, loc0, new ArrayList<>(0), session);
+    String t1 = SPARK_CATALOG + "." + SCHEMA_NAME + "." + DELTA_TABLE;
+    testTableReadWrite(t1, session);
+
+    session.sql(String.format("DELETE FROM %s WHERE i = 1", t1));
+    List<Row> rows = session.sql("SELECT * FROM " + t1).collectAsList();
+    assertThat(0 == rows.size());
+
+    session.stop();
+  }
+
+  @Test
+  public void testMergeDeltaTable() throws ApiException, IOException {
+    createCommonResources();
+    SparkSession session = createSparkSessionWithCatalogs(SPARK_CATALOG, CATALOG_NAME);
+
+    String loc0 = "s3://test-bucket0" + generateTableLocation(SPARK_CATALOG, DELTA_TABLE);
```

Comment:
nit: let's use `loc1` to match `t1`

Reply:
This is not addressed

Reply:
still not addressed...

Reply:
I messed up some local commits and a commit is not pushed....

Now it should be good.